catalog-rest-service/src/main/resources/json/schema/operations/pipelines/airflowPipeline.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$id": "https://open-metadata.org/schema/operations/pipelines/airflowPipeline.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Airflow Pipeline",
+  "description": "Airflow Pipeline is used to setup a DAG and deploy.",
```

Comment:
This entity will be only used for ingestion and operations required for OpenMetadata

And the other schema/data/entity/pipeline.json will be used for the typical pipeline assets as usual (even if they are airflow workflows that do ETL).

If yes, should we mention this in schema description to be explicit (although it is nested under operations)?

Reply:
done